Sources/Models/Views/StackView.swift

```diff
@@ -11,8 +11,10 @@ public struct StackView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let id: String
     public let elementClass: String = "UIStackView"
 
+    public let alignment: String?
```

Comment:
Why moving it. Please restore position to minimize the change and keep all views more copy pastable